x-pack/legacy/plugins/index_management/public/app/components/mappings_editor/components/load_mappings/load_mappings_provider.tsx

```diff
@@ -148,11 +148,18 @@ export const LoadMappingsProvider = ({ onJson, children }: Props) => {
   };
 
   const loadJson = () => {
-    const isValidJson = jsonContent.current!.validate();
+    if (jsonContent.current === undefined) {
```

Comment:
What do the changes in this file do? Are they germane to search?

Reply:
No, but I discovered a bug and fixed it here. Thanks for pointing it out though, we might need to port this change on a separate PR.